### ISSUE TITLE
Users with "Virtual YMCA" role see "Access Denied" on password reset attempt

### ISSUE THREAD

**Issue by froboy** [ymcatwincities/openy_gated_content#65]
**To reproduce:**
- Create a new user
- Add any of the member roles: "Virtual YMCA", "Virtual YMCA Trial", or "Virtual YMCA Premium"
- Visit `/user/password` and attempt a password reset with the email of the new account.

When a user with a member role attempts to reset their password via `/user/password` they are met with `403 Access Denied`. That could be confusing and does not give a path forward if the user unintentionally gets there.

**Proposed Resolution:**
Either create a custom 403 for those Roles or add some boilerplate help text to the 403 page.

**Comment by hamrant** [comment]
I think this was fixed in https://github.com/fivejars/openy_gated_content/pull/218

@froboy please review

**Comment by sarah-halby** [comment]
@froboy per @hamrant comment above, can you review the issue and determine if this has been fixed? If so, please close this item.

**Comment by hamrant** [comment]
This was postponed from release by @anpolimus. PR with fix ready for testing and review

**Comment by froboy** [comment]
I'll wait until that PR is merged. I tested https://sandbox-carnation-std-virtual-y.openy.org/user/login and I'm seeing a message, but it's only after the user goes back and reloads the form. See the attached video.


https://user-images.githubusercontent.com/238201/105095609-84be5a00-5a6b-11eb-92e9-da84ef1c5c61.mp4

Also, the error message could use some review:
> You're not allowing to reset password for user with Virtual Y role. 

could be

> If you are a Virtual YMCA member, please return to the home page to log in. Contact your branch for login help.

**Comment by sarah-halby** [comment]
@hamrant just confirming that this item is complete and ready to be closed?

**Comment by hamrant** [comment]
@sarah-halby yep, the fix was merged 4 days ago
